moveit_planners/stomp/example_pages.dox

```diff
@@ -0,0 +1,325 @@
+@endcode
+  - class: The class name
+*/
+
+/**
+@page polynomial_smoother_example PolynomialSmoother 
+Applies a polynomial to smooth out sharp turns
+@code
+  - class: stomp_moveit/PolynomialSmoother
+    poly_order: 5
+@endcode
+  - class:      The class name
+  - poly_order: The order to the polynomial.  Higher orders will allow for more turns
+*/
+
+/**
+@page trajectory_visualization_example TrajectoryVisualization 
+Publishes markers to visualize the current optimized trajectory
+@code
+  - class: stomp_moveit/TrajectoryVisualization
+    line_width: 0.02
+    rgb: [0, 191, 255]
+    error_rgb: [255, 0, 0]
+    publish_intermediate: True
+    marker_topic: stomp_trajectory
+    marker_namespace: optimized
+@endcode
+  - class:                The class name
+  - line_width:           The width of the line marker.
+  - rgb:                  The color of the line marker.
+  - error_rgb:            The color used when the trajectory is invalid.
+  - publish_intermediate: Whether to publish the optimized trajectory during each iteration.  When <b>False</b> only the
+                          final trajectory will be published.
+  - marker_array_topic:   The marker topic name.
+  - marker_namespace:     The namespace of the line marker.
+*/
+
+/**
+@page update_logger_example UpdateLogger 
+Stores the update values into files for post processing purposes. The file is compatible with the python numpy
+library and can be loaded into a numpy array by running.
+@code numpy.loadtxt('file_name.txt') @endcode
+
+The yaml configuration is as follows:
+@code
+- class: stomp_moveit/UpdateLogger
+  package: stomp_moveit
+  directory: log
+  filename: smoothed_update.txt
+@endcode
+  - class: The class name
+  - package: The ros package where the file will be saved
+  - directory:  The directory relative to the ros package
+  - filename:   The name of the file.
+*/
+
+/**
+@page constrained_cart_goal_example Constrained Cartesian Goal
+Modifies the trajectory update such that the goal of the updated trajectory is within the task manifold
+@code
+  update_filters:
+      - class: stomp_moveit/ConstrainedCartesianGoal
+        constrained_dofs: [1, 1, 1, 1, 1, 0]
+        cartesian_convergence: [0.005, 0.005, 0.005, 0.01, 0.01, 1.00]
+        joint_update_rates: [0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
+        max_ik_iterations: 100
+@endcode
+  - class:                  The class name.
+  - constrained_dofs:       Indicates which cartesians DOF are fully constrained (1) or unconstrained (0).  This vector is of the form
+                            [x y z rx ry rz] where each entry can only take a value of 0 or 1.
+  - cartesian_convergence:  A vector of convergence values for each cartesian DOF [vx vy vz wx wy wz].
+  - joint_update_rates:     The rates at which to update the joints during numerical ik computations.
+  - max_ik_iterations:      Limit on the number of iterations allowed for numerical ik computations.
+*/
+
```

Comment:
Did you reference any of this documentation when writing https://github.com/ros-planning/moveit_tutorials/pull/185?

Reply:
@davetcoleman  yes I addressed these [here](https://github.com/ros-planning/moveit_tutorials/pull/185/commits/cc156db6bfa3c9644d6539df095d9657255915cd#diff-c7e3ea56c76fbe54a22b97fdec63a393R178), [here](https://github.com/ros-planning/moveit_tutorials/pull/185/commits/cc156db6bfa3c9644d6539df095d9657255915cd#diff-c7e3ea56c76fbe54a22b97fdec63a393R184) and [here](https://github.com/ros-planning/moveit_tutorials/pull/185/commits/cc156db6bfa3c9644d6539df095d9657255915cd#diff-c7e3ea56c76fbe54a22b97fdec63a393R196) . I think I should change these links to belong to the new moveit link of [this file](https://github.com/raghavendersahdev/moveit/blob/stomp_planner/moveit_planners/stomp/example_pages.dox) once this gets merged and open a small new PR for this... Or alternatively I could just link it to [this file](https://github.com/ros-planning/moveit/tree/kinetic-devel/moveit_planners/stomp/example_pages.dox) right now. But this would cause travis to fail.